docs/index.rst

```diff
@@ -106,12 +109,15 @@ and some of the core features of Read the Docs.
    /config-file/index
    /webhooks
    /custom_domains
+   /canonical
```

Comment:
I'd also not consider this a core feature either. It feels much more like a guide, or a part of a larger piece of content around SEO. We already have this: https://docs.readthedocs.io/en/latest/guides/technical-docs-seo-guide.html#canonical-urls -- so I do think this feels more like guide content than a core feature. This could perhaps also better be part of the Custom Domain section, instead of it's own page?

Reply:
Any particular suggestions about SEO for titles? Sometimes I find a bit tedious to read "on Read the Docs" on every title, but perhaps it's good SEO-wise.

Or should I just read the guides we link at the bottom of that page? :)

Reply:
Regarding merging guides: I agree, just didn't want to do it on the first iteration. Will take care of it.

Reply:
Definitely not `on Read the Docs` for SEO. That's in the header already on every page. I think mostly thinking of terms people will search for. Probably more useful to note on specific pages, but thinking about 3-4 keywords and making sure they are in the page is definitely worth doing.

Reply:
Updated a few titles and filenames, although I think we can give a second pass at the SEO of our docs with some specialized tool in a second iteration?

Reply:
Yea, perhaps something to discuss in a meeting. We're working on this on the ads side, and knowing what keywords we care about is definitely helpful here. It's mostly just putting some thought into it.

Reply:
#8537